prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-tabs/site-registration-tabs.component.html

```diff
@@ -0,0 +1,33 @@
+<app-page [busy]="busy"
+          mode="full">
+
+  <ng-content select="app-page-header"></ng-content>
+
+  <ng-content select="app-page-subheader"></ng-content>
+
+  <mat-tab-group>
```

Comment:
Feels like we should consider disabling the transition animation between tabs.  We don't really have many transitions in the application, and this one is a bit busy - `animationDuration="0ms"`